### DISCUSSION TITLE
Docker base image is outdated

### DISCUSSION THREAD

**Discussion by osgger** [Koenkk/zigbee2mqtt#24279] [General]
Hi,

since

[https://github.com/Koenkk/zigbee2mqtt/commit/c16abdd83233c51fa5f7d6b56db6b19bea9b81f6](url)

we're stuck on Alpine 3.18.4 as base image. Could anyone ever figure it out, if Alpine is/was the reason for the issue mentioned in that commit?

Sister project zwave-js-ui [https://github.com/zwave-js/zwave-js-ui/blob/master/docker/Dockerfile](url) stays also with 3.18.4.

Does anybody know?

**Comment by Koenkk** [comment]
Later Alpine versions switch to Node 20 which we cannot use due to [this bug](https://github.com/npm/cli/issues/5721)
